src/Components/LogIn/LogIn.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from "react";
+import { useLogIn } from "../../Helpers/apiCalls";
+
+export default function LogIn(props) {
+    const { updateLogInRequest, users, setLoggedInUser } = props;
+
+    const [logInRequest, setLogInRequest] = useState({});
+
+    const { mutate: logIn } = useLogIn(setLoggedInUser);
+
+    function formHandler(event) {
+        updateLogInRequest(event, setLogInRequest);
+    }
+
+    return (
+        <form>
+            <h2>Log In</h2>
+            <span className="error"></span>
```

Comment:
we can probably remove this for now since we aren't using it for anything

Reply:
but I won't hold that up for the review. you can tell me to gfy and just merge it